src/aria2p/interface.py

```diff
@@ -577,6 +629,65 @@ def process_keyboard_event_main(self, event):
         elif event.key_code in Keys.UN_SELECT_ALL:
             pass  # TODO
 
+        elif event.key_code in Keys.MOVE_HOME:
+            if self.focused > 0:
+                self.focused = self.rows.index(self.rows[0])
+                logger.debug(f"Move focus home: {self.focused}")
+                if self.focused < self.row_offset:
+                    self.row_offset = self.focused
+                elif self.focused >= self.row_offset + (self.height - 1):
+                    self.row_offset = self.focused + 1 - (self.height - 1)
```

Comment:
Based on previous code, it seems this can happen when the screen height was shrinked. Could you copy/paste the same comment here as well? Because at first glance it seems like this condition is never met.